spyder/widgets/tests/test_editor.py

```diff
@@ -120,7 +121,7 @@ def test_find_number_matches(qtbot):
 
 def test_move_current_line_up(editor_bot):
     editor_stack, editor, qtbot = editor_bot
-        
```

Comment:
Please revert all blank removals you introduced in this file.

Reply:
Oops, accidentally left "remove trailing whitespace when saving" on...would be nice to have a toggle somewhere in the UI to make it less of a pain to be going in an out of preferences to keep changing/verifying it. Fortunately, it happened to be the last thing in my undo history so it has easy to revert.